docs/nav/nav-menu.yml

```diff
@@ -34,11 +34,19 @@
     content: 
     - item: Browser Aliases
       url: /testcafe/documentation/using-testcafe/common-concepts/browser-aliases
+    - item: Reporters
+      url: /testcafe/documentation/using-testcafe/common-concepts/reporters
 - folder: EXTENDING TESTCAFE
-  url: .  
+  url: /testcafe/documentation/extending-testcafe
   content:
-  - item: Reporter Plugins
-    url: .
+  - folder: Custom Reporter Plugin
+    url: /testcafe/documentation/extending-testcafe/custom-reporter-plugin
+    content:
+    - item: Creating Reporter Plugin
+      url: /testcafe/documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin
+      content:
```

Comment:
you must've forgotten to remove this `content:`
or do you mean this item is a folder?